src/Files.App/Dialogs/ReinstallPromptDialog.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<ContentDialog
```

Comment:
The encoding doesn't look right

Reply:
From UTF-8 with BOM(byte order mark) to UTF-8

Reply:
Done parhaps.